forms/HtmlEditorField.php

```diff
@@ -674,7 +680,9 @@ protected function getFieldsForOembed($url, $file) {
 					'center' => _t('HtmlEditorField.CSSCLASSCENTER', 'Centered, on its own.'),
 					'left' => _t('HtmlEditorField.CSSCLASSLEFT', 'On the left, with text wrapping around.'),
 					'right' => _t('HtmlEditorField.CSSCLASSRIGHT', 'On the right, with text wrapping around.')
-				)
+				),
+				Config::inst()->get('HtmlEditorField', 'media_alignment')
```

Comment:
I considered that but these settings can be a bit hard to find and I feel like people would be more inclined to look for this on HTMLEditorField rather than HtmlEditorField_Toolbar? `insert_width` is a very closely related setting so it would be nice for them to be grouped together on the base class I think.

BTW is there any way to view all the available config settings in SilverStripe? I feel like they should be exposed automatically in the API docs but the only way I know to find these kind of things is to go digging in the source code.

> Then you can use static::config()->get('media_alignment') and we won't have to manually rewrite the class name on merge to 4. :)

Ah, I wasn't aware of this shorthand. Thanks!